GraphWebApi/Controllers/GraphExplorerSnippetsController.cs

```diff
@@ -72,7 +72,7 @@ public async Task ExecuteResultAsync(ActionContext context)
         {
             context.HttpContext.Response.ContentType = "text/plain";
             var streamWriter = new StreamWriter(context.HttpContext.Response.Body);
```

Comment:
While at it, can we fix the disposal of the writer as well?

```suggestion
            using var streamWriter = new StreamWriter(context.HttpContext.Response.Body);
```

Reply:
Sounds like a good idea to me.